### PR TITLE
Reenable API tests that were accidentally skipped

### PR DESCRIPTION
**Pull request by Nutomic**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,7 +20,6 @@ query_testing/**/reports/*.json
 api_tests/node_modules
 api_tests/.yalc
 api_tests/yalc.lock
-api_tests/test.png
```
Comment (Nutomic) [inline .gitignore:23]: This isnt used anymore.

File: api_tests/src/shared.ts [hunk 2]
```diff
@@ -863,6 +863,7 @@ export function getCommentParentId(comment: Comment): number | undefined {
   if (split.length > 1) {
     return Number(split[split.length - 2]);
   } else {
+    console.log(`Failed to extract comment parent id from ${comment.path}`);
     return undefined;
```
Comment (Nutomic) [inline api_tests/src/shared.ts:867]: There are regular CI failures from this code, I want to figure out whats the reason.

https://woodpecker.join-lemmy.org/repos/129/pipeline/4397/21#L362

**Review by dessalines — APPROVED** [review]
Feel free to merge once you get it passing.

File: api_tests/src/shared.ts (hunk 2 above)
Comment (Nutomic) [reply api_tests/src/shared.ts:867]: [Here](https://woodpecker.join-lemmy.org/repos/129/pipeline/4410/21#L350) is one such failure (the previous one in line 347 is expected as part of the test. Im not familiar with the way comment paths work, but a path of `0` should probably never exist, right?

**Pull request merged**